app/src/main/java/com/dimelo/sampleapp/RcConfig.java

```diff
@@ -38,7 +45,43 @@ static Dimelo setupDimelo(Context context) {
         }
 
         dimelo.setDebug(true);
-        dimelo.setUserName("John Doe");
+
+        String userNameVal = RcConfig.getStringValueFromSharedPreference(context, RC_USER_NAME_VAL);
```

Comment:
I'll let you remove the other `Val`

```suggestion
        String userName = RcConfig.getStringValueFromSharedPreference(context, RC_USER_NAME_VAL);
```